website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -91,7 +91,7 @@ Use the Alert to communicate validation errors. For more details, refer to the [
   <A.Description>Correct the formatting of the following fields to update your user profile:
   </A.Description>
   <A.Description>
-  <Hds::Link::Inline @href="..." @color="secondary">Email address</Hds::Link::Inline>
+  <Hds::Link::Inline @href="#" @color="secondary">Email address</Hds::Link::Inline>
```

Comment:
I have mixed feelings about this (it's in a code snippet, the `...` are there to imply: "add your link here") but not a blocker.

Reply:
Added a meaningful title to PR along with Jira ticket number, great call.
I left the # for now, to follow the same pattern as the other links on the page vs ... 
If you feel strongly that we should use ... vs # then I will change them all for consistency in a fast follow PR.

Reply:
that's OK